src/main/java/com/powsybl/openloadflow/ac/equations/HvdcAcEmulationSide1ActiveFlowEquationTerm.java

```diff
@@ -24,9 +24,9 @@ public HvdcAcEmulationSide1ActiveFlowEquationTerm(LfHvdc hvdc, LfBus bus1, LfBus
     }
 
     private double p1(double ph1, double ph2) {
-        double rawP = rawP(p0, k, ph1, ph2);
-        double boundedP = boundedP(rawP);
-        return (isController(rawP) ? 1 : getVscLossMultiplier()) * boundedP;
+        double boundedP = boundedP(rawP(ph1, ph2));
+        // if converterStation1 is controller, then p1 is positive, otherwise it is negative
+        return isController(boundedP) ? boundedP : -getAbsActivePowerWithLosses(boundedP, lossFactor1, lossFactor2);
```

Comment:
Because isController() only checks the sign of the power transit. I thought there was no need to store rawP instead of directly checking the sign of boundedP.

Reply:
I think it is equivalent but as Didier is not here (these lines about boundedP are from him) maybe keep former lines.

Reply:
Yes it is equivalent (unless bounds of P are negative which they shouldn't). But I modified in order to keep the former lines.